collect/collect.go

```diff
@@ -67,16 +67,17 @@ var (
 )
 
 const (
-	descCollectDropped      = "Counter of dropped data points due to the queue being full."
-	descCollectSent         = "Counter of data points sent to the server."
-	descCollectQueued       = "Total number of items currently queued and waiting to be sent to the server."
-	descCollectAlloc        = "Total number of bytes allocated and still in use by the runtime (via runtime.ReadMemStats)."
-	descCollectGoRoutines   = "Total number of goroutines that currently exist (via runtime.NumGoroutine)."
-	descCollectPostDuration = "Total number of milliseconds it took to send an HTTP POST request to the server."
-	descCollectPostCount    = "Counter of batches sent to the server."
-	descCollectPostError    = "Counter of errors received when sending a batch to the server."
-	descCollectPostBad      = "Counter of HTTP POST requests where resp.StatusCode != http.StatusNoContent."
-	descCollectPostRestore  = "Counter of data points restored from batches that could not be sent to the server."
+	descCollectDropped           = "Counter of dropped data points due to the queue being full."
```

Comment:
@captncraig Can do that, but can that be added somewhere? style guide, contributing etc